lib/galaxy/jobs/runners/util/pykube_util.py

```diff
@@ -12,12 +12,15 @@
         Service,
         Ingress,
     )
+    from pykube.exceptions import HTTPError
 except ImportError as exc:
     KubeConfig = None
     Ingress = None
+    HTTPClient = None
```

Comment:
```suggestion
```
Since that is not used outside of the module I wouldn't provide a fallback

Reply:
It is exported by the module for use by kubernetes.py. Should I still remove it?
My IDE complains if it is not there.

Reply:
Are you sure about that ? I don't see it in kubernetes.py. If it is used there you should also add it to `__all__` below

Reply:
See https://github.com/galaxyproject/galaxy/pull/11715/files#diff-4b73c7a6546e8d68305e5e776cf411dc3e5db8a53c980a174ee330eaa22a1583R161

It is already included in __all__ https://github.com/galaxyproject/galaxy/pull/11715/files#diff-83b685d37c2473c87101dc9de4da0464b79bb2042539132e5c41ee7261e93b98R219

Reply:
I'm commenting on HTTPClient only, you seem to be linking to `HTTPError`

Reply:
Removed